onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -484,6 +484,28 @@ def aten_arange_start(
     return result
 
 
+@torch_op("aten::arange.start_step", private=True)
+def _aten_arange_start_step_onnx(
+    start: TRealUnlessFloat16OrInt8,
+    end: TRealUnlessFloat16OrInt8,
+    step: TRealUnlessFloat16OrInt8,
+    dtype: int,
+) -> Tuple[FLOAT, FLOAT, FLOAT]:
+    zero = op.Cast(0.0, to=FLOAT.dtype)
+    start = op.Cast(start, to=FLOAT.dtype)
```

Comment:
```suggestion
    start = op.Cast(start, to=FLOAT.dtype)
    step = op.Cast(step, to=FLOAT.dtype)
    end = op.Cast(end, to=FLOAT.dtype)
```

We can move all casts to the top for clarity